src/components/Table/table-cells/TableBodyCell.tsx

```diff
@@ -27,7 +28,7 @@ export const TableBodyCell: React.FC<TableCellProps> = ({
     // TODO: When no iconStart is set we should just render children to avoid too nested HTML elements
     // TODO: Add iconEnd somewhere or remove it from the props, it seems we never use it
     const renderCellContent = () => (
-        <IconTextIcon component="div" iconStart={iconStart}>
+        <IconTextIcon component="div" iconStart={iconStart} typographyProps={{className: clsx('moonstone-tableCellContent', 'flexFluid')}}>
```

Comment:
I think it's better to add a prop to manage if the cell is scrollable or not. The default behavior must still be to truncate the text when it's too long.
In that case, we can rename the CSS class by something more descriptive like `moonstone-tableBodyCell_scrollable`.

Reply:
👍